### PR TITLE
Extending cruise ship data

### PR DESCRIPTION
**Pull request by stevemanion**
Putting together cruise ship data for cruises that disembarked passengers in New Zealand close to the cruise ship shut down deadline, in particular those that had known infection cases, as well as overseas cruises where New Zealanders were repatriated. Will continue to extend this, as there was apparently up to 23 cruise ships with New Zealanders on them at the time of shut down (see https://www.rnz.co.nz/news/national/413459/covid-19-more-than-70-new-zealanders-still-stuck-on-cruise-ships). Once this is done, we can begin acquiring cruise ship track data for these ships.

Also added the current cruise ship schedule for New Zealand from/to July 2019/2020, this was found at: https://newzealandcruiseassociation.com/2019-20-cruise-ship-schedule/.

With @kche309's help, hopefully we can get the cruise-track data for this work. Also tagged @whistlebee as an observer to see what overlap their may be with the flight-track and cruise-track data.

### CODE REVIEW

**Comment by stevemanion** [comment]
⛵ FYI @kche309, @whistlebee have finished putting together cruise ship information. In order of priority, the three main groups to now try extend data for are:

Group 1 : Infected ships that docked in New Zealand:
* Ruby Princess
* Celebrity Solstice
* Ovation Of The Seas

Group 2: Infected ships that New Zealand citizens were repatriated from:
* Diamond Princess
* Coral Princess
* Costa Pacifica
* Greg Mortimer
* MS Rotterdam
* MS Zaandam

Group 3: Ships suspected of infection that docked in New Zealand
* Golden Princess
* MSC Magnifica

The remaining ships are also of interest as New citizens were repatriated from them or they docked in New Zealand a month leading up to the ban on cruise ships - however since they do not have any reported infections, they are low priority for now.

@kche309 ready to merge if you are happy 😄

File: data/sources/nz-cruise-association/nz-cruise-schedule.csv [hunk 1]
```diff
@@ -0,0 +1,1124 @@
+Ship,Port,Arrival,Arrival time,Departure,Departure time,Guests (Lower Berths),Crew
```
Comment (whistlebee) [inline data/sources/nz-cruise-association/nz-cruise-schedule.csv:1]: Different column naming style to the other file

File: data/sources/nz-cruise-association/nz-cruise-schedule.csv [hunk 2]
```diff
@@ -0,0 +1,1124 @@
...
+Pacific Aria,Auckland,05/08/19,07:00:00,05/08/19,16:00:00,1260,597
+Sea Princess,Auckland,25/08/19,10:00:00,25/08/19,21:00:00,2016,839
+Sea Princess,Bay Of Islands,26/08/19,08:00:00,26/08/19,17:00:00,2016,839
+Sea Princess,Auckland,29/09/19,10:00:00,29/09/19,20:00:00,2016,839
+Sea Princess,Bay Of Islands,30/09/19,07:00:00,30/09/19,16:00:00,2016,839
+Maasdam,Fiordland (cancelled),30/09/19,08:00:00,30/09/19,17:00:00,1258,580
+Maasdam,Dunedin,01/10/19,10:00:00,01/10/19,18:00:00,1258,580
+Maasdam,Lyttelton,02/10/19,08:00:00,02/10/19,17:00:00,1258,580
+Majestic Princess,Fiordland,03/10/19,07:00:00,03/10/19,16:00:00,3560,1357
+Maasdam,Kaikoura (cancelled),03/10/19,08:00:00,03/10/19,17:00:00,1258,580
```
Comment (whistlebee) [inline data/sources/nz-cruise-association/nz-cruise-schedule.csv:15]: Can we have the status as a separate column?

File: data/sources/nz-cruise-association/nz-cruise-schedule.csv (hunk 1 above)
Comment (stevemanion) [reply data/sources/nz-cruise-association/nz-cruise-schedule.csv:1]: @whistlebee good catch, shallChangeToCamelCase 😆

File: data/sources/nz-cruise-association/nz-cruise-schedule.csv (hunk 2 above)
Comment (stevemanion) [reply data/sources/nz-cruise-association/nz-cruise-schedule.csv:15]: @whistlebee Yep, again with your spot of the headers not in camelCase, we can put together a richer version of the NZ Cruise Schedule. I assume you mean to separate the `(cancelled)` to its own column? Also the dates need to be brought into line with the `dd/mm/yyyy` format. I'll do this now 😄

File: data/sources/nz-cruise-association/nz-cruise-schedule.csv (hunk 1 above)
Comment (kche309) [reply data/sources/nz-cruise-association/nz-cruise-schedule.csv:1]: Sounds good, agree to keeping the source version intact

File: data/sources/nz-cruise-association/nz-cruise-schedule.csv (hunk 2 above)
Comment (stevemanion) [reply data/sources/nz-cruise-association/nz-cruise-schedule.csv]: @whistlebee requested changes now committed, a new cleaned up cruise ship schedule can now be found in `data/meta`

**Pull request merged**